test/extensions/quic_listeners/quiche/active_quic_listener_config_test.cc

```diff
@@ -41,6 +48,35 @@ TEST(ActiveQuicListenerConfigTest, CreateActiveQuicListenerFactory) {
   EXPECT_EQ(2000u, quic_config.IdleNetworkTimeout().ToMilliseconds());
   // Default value if not present in config.
   EXPECT_EQ(20000u, quic_config.max_time_before_crypto_handshake().ToMilliseconds());
+  envoy::config::core::v3::RuntimeFeatureFlag& runtime_enabled =
+      ActiveQuicListenerFactoryPeer::runtimeEnabled(
+          dynamic_cast<ActiveQuicListenerFactory&>(*listener_factory));
+  EXPECT_EQ(true, runtime_enabled.default_value().value());
+  EXPECT_EQ("foo_key", runtime_enabled.runtime_key());
+}
+
+TEST(ActiveQuicListenerConfigTest, QuicListenerFlagNotConfigured) {
+  std::string listener_name = QuicListenerName;
+  auto& config_factory =
+      Config::Utility::getAndCheckFactoryByName<Server::ActiveUdpListenerConfigFactory>(
+          listener_name);
+  ProtobufTypes::MessagePtr config = config_factory.createEmptyConfigProto();
+
+  std::string yaml = R"EOF(
+    max_concurrent_streams: 10
+    idle_timeout: {
+      seconds: 2
+    }
+  )EOF";
+  TestUtility::loadFromYaml(yaml, *config);
+  Network::ActiveUdpListenerFactoryPtr listener_factory =
+      config_factory.createActiveUdpListenerFactory(*config, /*concurrency=*/1);
+  EXPECT_NE(nullptr, listener_factory);
+  envoy::config::core::v3::RuntimeFeatureFlag& runtime_enabled =
+      ActiveQuicListenerFactoryPeer::runtimeEnabled(
+          dynamic_cast<ActiveQuicListenerFactory&>(*listener_factory));
+  EXPECT_EQ(false, runtime_enabled.default_value().value());
```

Comment:
Can you check runtime_enabled.has_default_value() here instead? If it's false, we know that FeatureFlags will be instantiated with default value true.

Reply:
fixed